src/main/java/org/commcare/formplayer/services/MediaHandler.kt

```diff
@@ -54,6 +75,12 @@ class MediaHandler(val file: MultipartFile) {
     }
 
     fun cleanMedia(parentDirPath: Path, fileId: String): Boolean {
+        try {
```

Comment:
we should delete the meta data record after delete from filesystem is successful.